artifacts/checksums.txt

```diff
@@ -1,3 +1,4 @@
-b8b8650aa62de7172b201b2a8e4e9df37757cb634dbf2e4a7f9b9e353ba5ea46  ibc_transfer.wasm
-a0c59cc8b9946737a36482120308f5fed541bc8012e474d08e24c1009f41751f  neutron_interchain_queries.wasm
-f749ad20727695ec8745dbdf7a23b818cff0b36aeee4be3381192c490be12943  reflect.wasm
+109eb698a4a361767917b2b4eb404d792f01ff80a6e84653867a94f5f935c0fe  ibc_transfer.wasm
+182581564b17ff4c9baff602e0beda0efba438e69e9f28fd531bd26ec5707cfd  neutron_interchain_queries.wasm
+819945933c6935cecce7be7f2a2f3911fdd253fe33ad2079e9ea3151dece9dce  neutron_interchain_txs.wasm
+92a6fca23a6ea41ef4bb3ade8b3543602e1fb1c83e2a34b054e3afbed40e39df  reflect.wasm
```

Comment:
have you pushed the latest version of these files? my make build has made changes to the result .wasm files

Reply:
same for me actually. I get these hashes:
```
e2a0010c52831b3181cdee29a543adc5f608a8460a8065f68f71029d22b0bbe0  ibc_transfer.wasm
623eacb39b51775c9f6637c80cba3470d5e17a946bde9805ea8d27d4fb2b6f2c  neutron_interchain_queries.wasm
693aa6021ccbf11fbd46fff005af871e56fa4f78d14bd5fa85acbce4a98b6b8a  neutron_interchain_txs.wasm
cf66ca7c6f7027538671f1c0ea547f35f71dbfcc50907bc0bfb41a0782950dab  reflect.wasm
```

Reply:
please verify now